SmartDeviceLinkTests/DevAPISpecs/SDLVoiceCommandUpdateOperationSpec.m

```diff
@@ -244,6 +250,15 @@ @interface SDLVoiceCommand()
             });
         });
     });
+
+    // should test voiceCommandsInArray and return the difference between two arrays
+    it(@"should have a priority of 'normal'", ^{
+        testOp = [[SDLVoiceCommandUpdateOperation alloc] init];
+        NSArray<SDLVoiceCommand *> *voiceCommands = [testOp voiceCommandsInArray:@[newVoiceCommand1, newVoiceCommand2] notInSecondArray:@[newVoiceCommand1]];
+
+        expect(voiceCommands).to(haveCount(1));
+        expect(voiceCommands.firstObject).to(equal(newVoiceCommand2));
+    });
```

Comment:
This should not be the only test added (also the title is wrong).

You should add a test to set up the voice command update operation with (1) a pending array identical to the old array, (2) with the same but fewer so that it should only delete, (3) with the same and a few more so that it should only add, (4) with completely different so that it deletes and adds.